### PR TITLE
updated dependency section of the README.md and linked to it in the 'Required dependencies' section of the 'Getting Started' guide

### PR DESCRIPTION
**Pull request by bamling**
## Description

I updated the **Dependencies** section of the _README.md_ and added a few other, popular Linux distributions. The new list should be as accurate as possible - I booted up a fresh image of each distribution in a docker container to verify that the amethyst tools can be installed and a simple project created with `amethyst new <name>` can compile.

I also figured a direct link to the _README.md_ in the books **Required dependencies** section of the **Getting Started** guide was a better approach to having a rough list of required dependencies listed there. This way we only have to keep one place up to date.

## PR Checklist

By placing an x in the boxes I certify that I have:

- [x] Ran `cargo test --all` locally if this modified any rs files.
- [x] Ran `cargo +stable fmt --all` locally if this modified any rs files.
- [x] Updated the content of the book if this PR would make the book outdated.
- [x] Added a changelog entry if this will impact users, or modified more than 5 lines of Rust that wasn't a doc comment.
- [x] Added unit tests for new APIs if any were added in this PR.
- [x] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,23 +108,35 @@ If you are compiling on Linux, make sure to install the dependencies below.
...
 ```
-$ sudo apt install pkg-config libasound2-dev libssl-dev cmake libfreetype6-dev libexpat1-dev libxcb-composite0-dev
+$ apt-get install gcc pkg-config openssl libasound2-dev cmake build-essential python3 libfreetype6-dev libexpat1-dev libxcb-composite0-dev
 ```
 
 ### Fedora
 
 ```
-$ sudo dnf install alsa-lib-devel openssl-devel cmake freetype-devel expat-devel libxcb-devel
+$ yum install gcc openssl-devel alsa-lib-devel cmake make gcc-c++ freetype-devel expat-devel libxcb-devel
```
Comment (torkleyy) [inline README.md]: Why did you change that to `yum`?

Comment (torkleyy) [reply README.md]: I think we're also missing `pkgconfig`.

File: README.md [hunk 2]
```diff
@@ -108,23 +108,35 @@ If you are compiling on Linux, make sure to install the dependencies below.
...
+
+```
+$ pacman -Sy grep gcc pkgconfig openssl alsa-lib cmake make python3 freetype2 awk libxcb
+```
+
 ### Debian/Ubuntu
 
 ```
-$ sudo apt install pkg-config libasound2-dev libssl-dev cmake libfreetype6-dev libexpat1-dev libxcb-composite0-dev
+$ apt-get install gcc pkg-config openssl libasound2-dev cmake build-essential python3 libfreetype6-dev libexpat1-dev libxcb-composite0-dev
```
Comment (jojolepro) [inline README.md]: its more recommended to use apt than apt-get

Comment (torkleyy) [reply README.md]:
```suggestion
# apt install gcc pkg-config openssl libasound2-dev cmake build-essential python3 libfreetype6-dev libexpat1-dev libxcb-composite0-dev
```

File: README.md (hunk 1 above)
Comment (torkleyy) [reply README.md]:
```suggestion
# dnf install pkgconfig gcc openssl-devel alsa-lib-devel cmake make gcc-c++ freetype-devel expat-devel libxcb-devel
```

File: README.md [hunk 3]
```diff
@@ -108,23 +108,35 @@ If you are compiling on Linux, make sure to install the dependencies below.
...
 
 ```
-$ sudo dnf install alsa-lib-devel openssl-devel cmake freetype-devel expat-devel libxcb-devel
+$ yum install gcc openssl-devel alsa-lib-devel cmake make gcc-c++ freetype-devel expat-devel libxcb-devel
+```
+
+### openSUSE
+
+```
+$ zypper install gcc pkg-config libopenssl-devel alsa-devel cmake gcc-c++ python3 freetype2-devel libexpat-devel libxcb-devel
```
Comment (torkleyy) [inline README.md]:
```suggestion
# zypper install gcc pkg-config libopenssl-devel alsa-devel cmake gcc-c++ python3 freetype2-devel libexpat-devel libxcb-devel
```

**Comment by bamling** [comment]
bors r=jojolepro,mralve

**Comment by torkleyy** [comment]
Thank you @bamling!
bors r+
